client.go

```diff
@@ -419,18 +422,24 @@ func (c *Client) UnlockCommits(vsys, admin string) error {
 // Commits result in a job being submitted to the backend.  The job ID and
 // if an error was encountered or not are returned from this function.  If
 // the job ID returned is 0, then no commit was needed.
-func (c *Client) Commit(desc string, dan, pao, force, sync bool) (uint, error) {
+func (c *Client) Commit(desc string, members []string, dan, pao, force, sync bool) (uint, error) {
```

Comment:
Can you change the variable name from "members" to "admins" to make it clearer what it's for?

Reply:
Updated.